source/assets/javascripts/application.js

```diff
@@ -20,3 +20,9 @@ if (waypointElement !== null) {
 if (/iPad|iPhone|iPod/.test(navigator.userAgent)) {
   document.documentElement.className += " ios-device";
 }
+
+jQuery.get("http://ipinfo.io", function(response) {
+  if (response.country == 'ZA') {
+    $("#London").insertAfter("#CapeTown");
+  }
+}, "jsonp");
```

Comment:
It will be a tiny JS script as well so we can probably get away with including the JS directly in the bottom of the contact page instead of having to check for an element's existence similar to the waypoint toggle.